testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java

```diff
@@ -91,7 +92,8 @@ public void addAdapterTestRealms(List<RealmRepresentation> testRealms) {
 
     @Override
     protected boolean isImportAfterEachMethod() {
-        return false;
+        // For proper cleanup of test methods
```

Comment:
This test needs proper clean-up as all test cases set time offsets and for app server and auth server who live in different JVM is a little bit more challenging.